pkg/config/config_template.yaml

```diff
@@ -51,6 +51,13 @@ api_key:
 # Warning: percentiles must be specified as yaml strings
 #
 # histogram_percentiles: ["0.95"]
+#
+# Copy histgram values to distributions for true global distributions (in beta)
+# This will increase the number of custom metrics created
+# histogram_copy_to_distribution: true
```

Comment:
nit: as a convention, unless explained explicitly in the comment, the commented-out value (`true` here) should reflect the default value of the option.
Could you either explicitly mention `(default: false)` in the inline comment or set `false` as the commented-out value?